perf/benchmark/configs/istio/security_authz/prerun.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "Configure Security Policies..."
+POLICY_PATH="${WD}/security/generate_policies"
+
+echo "Build Security Policy Generator..."
```

Comment:
maybe we should just run the generator offline one time and check `largeSecurityPolicy.yaml` into the repo. Then its a bit simpler and we know it will not change between runs?